components/dashboard/src/App.tsx

```diff
@@ -479,6 +480,9 @@ function App() {
         url.pathname = "/";
         window.location.replace(url);
         return <div></div>;
+    } else if (window.location.pathname === "/components/display") {
+        // } else if (window.location.pathname === "/components/display" && process.env.NODE_ENV !== "production") {
```

Comment:
Good to remove? 

```suggestion
"production") {
```

Reply:
It's a page for @gtsiolis to view the UX of component `Alert` will remove this router before unhold this PR

Reply:
For visibility, it's about the following page which is available through this PR:

> https://hw-fix-8028.staging.gitpod-dev.com/components/display

Cc @loujaybee because https://github.com/gitpod-io/gitpod/pull/8783#discussion_r838451420
CC @geropl because https://github.com/gitpod-io/gitpod/pull/8783#issuecomment-1091747783

Reply:
FYI: Capture of display page is on PR description, removed it in code